src/main/webapp/app/modules/home/activity-element.tsx

```diff
@@ -1,18 +1,34 @@
 import './activity-element.scss';
 
 import React from 'react';
+import axios from 'axios';
 import { Row, Col, Card, CardText, CardBody, CardTitle, CardGroup } from 'reactstrap';
-import { Translate, TextFormat } from 'react-jhipster';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Translate, translate, TextFormat } from 'react-jhipster';
 import { APP_DATE_FORMAT } from 'app/config/constants';
+import HideRecordButton from 'app/shared/layout/hide-record-button';
+import { toast } from 'react-toastify';
 
 const ActivityElement = props => {
   const maxConflicts = 3;
   const conflictsToDisplay = props.activity.conflicts.slice(0, maxConflicts);
   const areAllDisplayed = props.activity.conflicts.length <= maxConflicts;
 
+  const hideActivity = event => {
+    const matchIds = props.activity.organizationMatches;
```

Comment:
you should add the new field to the activity model

Reply:
fixed